### DISCUSSION TITLE
Deployment Disabled?

### DISCUSSION THREAD

**Discussion by levinunnink** [vercel/community#1922] [Help]
### Summary

Vercel has suddenly been randomly disabling my production deployment without warning. Redeploying the site restores it. I've been running this site for almost two years and now it's become unstable. 

<img width="608" alt="image" src="https://user-images.githubusercontent.com/788141/228830580-c05ac97c-dfdb-436c-b777-d66f39bc165c.png">

What is causing this?

**Comment by sumedh1996** [comment]
@levinunnink Any updates on this issue?

**Comment by amyegan** [comment]
If you're still seeing this error, please let me know the URL or ID of the deployment that is throwing this error. That will help me find more information about what is causing it 🙏

**Comment by levinunnink** [comment]
@amyegan Update on this: It appears that one of my users had created a form that was phishing. I understand the need to disable malicious deployments but Vercel did this without any notification to me or warning. I just woke up one morning and my whole app was down and users were complaining. Does Vercel honestly not even give a warning or notification when it disables a deployment?

**Reply by amyegan** [reply]
Thanks for sharing this update, Levi. Your feedback about the disabled deployment notification was shared with the team. Given the sensitive nature of the topic, please follow up with the helpdesk case if you have any further questions.
